security/ThreadModel.md

```diff
@@ -0,0 +1,9 @@
+# What has happend?
```

Comment:
great answers .... this happens too often with old security of places like hospitals.  This is true even when they have important information like PMI with HIPAA laws and penalties but they don't upgrade either knowingly or unknowingly until its too late and data is compromised.